### PR TITLE
Handle wishlist functionality

### LINKED ISSUE #8: Todo: handle_cart_functionality
The main focus of the issue is:

- Handle functionality like add to cart, remove from the cart.
- Handle all the calculation part.

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
This pull request consists of 

- Add to wishlist functionality 
- Login and signup pages 
- minor UI changes

### CODE REVIEW

File: src/components/ProductCard.jsx [hunk 1]
```diff
@@ -32,7 +48,7 @@ const cartData = { _id:_id,
         </div>
         <div className="card-badge card-badge-position">{category}</div>
         <div className="favourite-icon favourite-icon-position">
-          <a className="btn btn-link-noBorder" href>
+          <a className="btn btn-link-noBorder" href="#" onClick={()=> addToWish(wishData)}>
```
Comment (sadathshariff) [inline src/components/ProductCard.jsx]: any reason for using a tag here

**Review by sadathshariff — APPROVED** [review]
Overall the code looks good

File: src/pages/Wishlist.jsx [hunk 2]
```diff
@@ -1,7 +1,38 @@
...
\ No newline at end of file
+  return (
+    <div className="Page-container">
+      <div className="section-heading flex-center p-1 font-lg">
+        My Wishlist({wishBasket.length})
+      </div>
+      
+        <div className="wish-grid">
+          {wishBasket.map((cartItem) => {
+            console.log(cartItem)
```
Comment (aishwarya-bm) [inline src/pages/Wishlist.jsx]: You can destructure the item here and use its properties.
Remove console like logs

**Review by aishwarya-bm — APPROVED** [review]
Looks all good , clean code . Left a small suggestion

File: src/components/ProductCard.jsx (hunk 1 above)
Comment (vaibhav-mahalle) [reply src/components/ProductCard.jsx]: It was not intentional I will make the changes right away

File: src/pages/Wishlist.jsx (hunk 2 above)
Comment (vaibhav-mahalle) [reply src/pages/Wishlist.jsx]: thank you for pointing that out will make the changes.

**Pull request merged**